src/common/src/util/epoch.rs

```diff
@@ -150,20 +156,43 @@ impl EpochPair {
         Self { curr, prev }
     }
 
-    pub fn inc(&mut self) {
-        self.curr += 1;
-        self.prev += 1;
+    pub fn inc_for_test(&mut self) {
+        self.prev = self.curr;
+        self.curr += EPOCH_INC_MIN_STEP_FOR_TEST;
     }
 
-    pub fn inc_for_test(&mut self, inc_by: u64) {
-        self.prev = self.curr;
+    pub fn new_test_epoch(curr: u64) -> Self {
```

Comment:
How to test the real scenarios if we only use test epoch in UT ?

Reply:
I don't quite understand what you mean, after this pr, we can test mem table spill in unit test.